pkg/referrerstore/oras/authprovider/authprovider.go

```diff
@@ -127,6 +128,7 @@ func (d *defaultAuthProvider) Provide(ctx context.Context, artifact string) (Aut
 	authConfig := AuthConfig{
 		Username: dockerAuthConfig.Username,
 		Password: dockerAuthConfig.Password,
+		IdentityToken: dockerAuthConfig.IdentityToken,
```

Comment:
I wonder if we could just overwrite `Password` with `dockerAuthConfig.IdentityToken` if it's not empty? Then we don't need to make the change in `oras.go`.

Reply:
Wondered the same, but didn't know if that would have any side effects. Will close this in favor of using `RefreshToken`